source/features/prevent-pr-merge-panel-opening.tsx

```diff
@@ -0,0 +1,36 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+let controller: AbortController;
+
+async function sessionResumeHandler(): Promise<void> {
+	await Promise.resolve(); // The `session:resume` event fires a bit too early
+	const cancelMergeButton = select('.merge-branch-form .js-details-target');
+	if (cancelMergeButton) {
+		cancelMergeButton.click();
+		controller.abort();
+	}
```

Comment:
Several `session:resume` events are fired upon page load, each with the same metadata, so (afaict) they're virtually indistinguishable.

From what I can tell the merge panel DOM will only be injected after the second one, hence the check and the abort controller (since we can't just use `{once: true}` in `addEventListener`).